cryptnono/values.yaml

```diff
@@ -51,6 +51,11 @@ detectors:
     enabled: true
     resources: {}
     configs: {}
+    metrics:
+      enabled: false
+      port: 12121
+      # Add prometheus annotations to the pods
+      prometheusScrape: true
```

Comment:
`prometheusScrape` is just for convenience since mybinder.org uses Prometheus, if we want to keep this generic then we could remove this and require the user to set `Values.podAnnotations` themselves.

Reply:
yea i think this is fine!